app/api/v1/endpoints/connectors.py

```
@@ -0,0 +1,61 @@
+
+def _health_payload(connectors: list, status: str) -> dict:
+    """Build response body; serialize ConnectorHealth for JSON (datetime → iso)."""
+    out = []
+    for h in connectors:
+        out.append({
+            "connector_id": h.connector_id,
+            "status": h.status.value if isinstance(h.status, ConnectorStatus) else h.status,
+            "last_event_at": h.last_event_at.isoformat() if h.last_event_at else None,
+            "error": h.error,
+            "lag": h.lag,
+        })
```

Ruff (0.15.5)

[warning] Use a list comprehension to create a transformed list

(PERF401)